tests/index.html

```diff
@@ -90,6 +90,7 @@
       <header class='header'>
         <div id='nav'>
             <a href='../'>Samples</a>
+            <a href='../layers-samples/'>Layers Samples</a>
             <!--a href='../proposals/'>Proposals</a-->
```

Comment:
```suggestion
            <a href='../proposals/'>Proposals</a>
```

I have no idea why this was commented out before. It's not a problem you introduced, but while you're touching this file it would be nice to fix it.